webpack.config.js

```diff
@@ -20,7 +20,7 @@ var loaders = [
     }
   },
   {
-    "test": /\.html?$/,
+    "test": /(\.html?|\.md)$/,
```

Comment:
This should probably just be `.md`. I originally built this in webpackbin which doesn't let you add .md files, only .html files, which is why this was originally built this way.